profiles/models.py

```diff
@@ -140,6 +140,15 @@ def __str__(self):
 class ProgramCertificate(models.Model):
     """An external model that syncs with data from our data platform"""
 
+    # an external hash that hightouch uses for the sync
+    record_hash = models.CharField(
```

Comment:
This needs to be treated as the primary key field for the table. There isn't any semantic utility in maintaining the auto-generated `id` field for this table

Reply:
made row_hash the primary key and dropped id